README.adoc

```diff
@@ -100,11 +100,11 @@ Each https://dashboard.internetcomputer.org/releases[release proposal] includes
 
 To verify the build reproducibility of a specific `Elect Replica` proposal, you can just copy the one liner below to a fresh Ubuntu VM 22.04 or higher:
 
-    $ sudo apt-get install -y curl && curl --proto '=https' --tlsv1.2 -sSLO https://raw.githubusercontent.com/dfinity/ic/master/gitlab-ci/tools/repro-check.sh && chmod +x repro-check.sh && ./repro-check.sh -p <proposal>
+    $ sudo apt-get install -y curl && curl --proto '=https' --tlsv1.2 -sSLO https://raw.githubusercontent.com/dfinity/ic/master/ci/tools/repro-check.sh && chmod +x repro-check.sh && ./repro-check.sh -p <proposal>
```

Comment:
Would you be okay if we merge this after the RC cut today? Otherwise we will have a problem with the NNS proposal summaries.